routes/product/helpers.js

```diff
@@ -12,28 +17,12 @@ module.exports = {
         }
     },
     productCreate: async (req, res) => {
-        const {
-            name,
-            quantity,
-            producerId,
-            typeOfProduct,
-            createdAt,
-            updatedAt,
-            deletedAt,
-        } = req.body;
-
-        const product = Object.assign( new Product(), {
-            createdAt,
-            deletedAt,
-            name,
-            producerId,
-            quantity,
-            typeOfProduct,
-            updatedAt,
-        });
-
         try {
+            const productData = _pick(req.body, CREATE_KEYS);
```

Comment:
Wow, this is more than clean.
please, what is happening here?

Reply:
clean but still makes intuitive sense